src/lib/popover/popover.spec.ts

```diff
@@ -1068,13 +1107,34 @@ describe('SatPopover', () => {
 });
 
 /**
- * This component is for testing that passing an invalid popover
- * to an anchor will throw an error.
+ * This class is used to allow accessing private API from
+ * within tests in a type safe manner. It is only used for
+ * typings.
+ */
+// @ts-ignore typescript doesn't like that we made prop private -> public
+export class OpenAPISatPopover extends SatPopover {
+  _anchoringService: SatPopoverAnchoringService;
```

Comment:
I created this class to allow accessing a SatPopovers `_anchoringService` for tests. A popover can be typed using this class to allow accessing the `_anchoringService` without errors.